internal/service/alarms/internal/db/migrations/000006_create_alarm_service_configuration_table.up.sql

```diff
@@ -0,0 +1,26 @@
+-- Holds information about the alarm service configuration
+CREATE TABLE IF NOT EXISTS alarm_service_configuration (
+    -- O-RAN
+    retention_period INT NOT NULL, -- Number of days for alarm history to be retained.
+    extensions JSONB, -- Additional data for extensibility
+
+    -- Internal
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(), -- Unique identifier for each alarm service configuration
+    created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP, -- Record creation timestamp
+    updated_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP -- Record last update timestamp
+);
+
+-- Trigger function to update updated_at timestamp for alarm_service_configuration
+CREATE OR REPLACE FUNCTION update_alarm_service_configuration_timestamp()
+    RETURNS TRIGGER AS $$
+BEGIN
+    NEW.updated_at = CURRENT_TIMESTAMP;
```

Comment:
Can't we do this simply in code rather than rely on a trigger?

Reply:
We took this convention for alarm server. At least, for the update_at field. If we are gonna change it, then we will need another pr to modify all the tables